### DISCUSSION TITLE
Center PWA Logo based off Inner Circle instead of Feather

### DISCUSSION THREAD

**Discussion by Dual-0** [danny-avila/LibreChat#4434] [Feature Requests & Suggestions]
### What happened?

Hello,

I notice the logo is not centered. The PWA Logo on my Pixel is not so nice.

### Steps to Reproduce

Install LibreChat as PWA

### What browsers are you seeing the problem on?

Mobile (Android)

### Screenshots

![Screenshot_20240918-192205](https://github.com/user-attachments/assets/1e46d510-dbe7-4068-bb63-d59215a49788)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
I believe the logo is centered based off the feather, maybe @fuegovic can comment better on this. On iPhone/Windows Desktop/macOS, this looks fine and in line with other apps, so I'm moving this to suggestions.

**Reply by fuegovic** [reply]
The alignment should be centered according to the overall shape. Since I don't use Android currently, perhaps @berry-13 could determine if it's an issue on Android in general.

![image](https://github.com/user-attachments/assets/7b0f32db-eba2-4e43-a068-e9ec16899e8d)

**Reply by berry-13** [reply]
yes I'm also getting this issue but only on Chrome. Brave and Internet (Samsung) don't have any issue
